tools/cli/internal/changelog/merge.go

```diff
@@ -0,0 +1,323 @@
+package changelog
+
+import (
+	"encoding/json"
+	"fmt"
+	"sort"
+
+	"github.com/mongodb/openapi/tools/cli/internal/changelog/outputfilter"
+	"github.com/tufin/oasdiff/checker"
+)
+
+const (
+	endpointAddedCode       = "endpoint-added"
+	endpointDeprecatedCode  = "endpoint-deprecated"
+	endpointReactivatedCode = "endpoint-reactivated"
+	notSetPriority          = 10
+	changeTypeRelease       = "release"
+	changeTypeUpdate        = "update"
+	changeTypeDeprecated    = "deprecate"
+	changeTypeRemove        = "remove"
+	ChangeLevelErr          = 3
```

Comment:
nit - are we able to import this from oasdiff? (once they changed it and it messed a bunch of stuff)

Reply:
updated